app/containers/ItunesProvider/DetailContainer/index.js

```diff
@@ -0,0 +1,160 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { get } from 'lodash';
+import { Card, Skeleton } from 'antd';
+import styled from 'styled-components';
+import { injectIntl } from 'react-intl';
+import { injectSaga } from 'redux-injectors';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { selectSongContainer, selectSongsData, selectSongsError, selectLoading } from '../selectors';
+import { songContainerCreators } from '../reducer';
+import { songContainerSaga } from '../saga';
+import T from '@components/T';
+import LazyImage from '@app/components/LazyImage';
+import AudioPlayer from '@app/components/AudioPlayer';
+
+const { Meta } = Card;
+
+const LinkButton = styled.a`
+  && {
+    margin-top: 1em;
+    padding: 14px 25px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    background-color: rgba(0, 0, 0, 0.45);
+    color: #fff;
+  }
+`;
+
+const TextContainer = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+  }
+`;
+
+const ContentContainer = styled.div`
+  && {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    max-width: ${props => props.maxwidth}px;
+    padding: ${props => props.padding}px;
+  }
+`;
+const ResultContainer = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    max-width: 80vw;
+    width: 100%;
+    margin: 20px auto;
+    padding: ${props => props.padding}px;
+  }
+`;
+
+export function DetailContainer({
+  dispatchTrack,
+  intl,
+  songsData = {},
```

Comment:
use defaultProps instead of setting default values here

Reply:
okay, makes sense